packages/dropdown/src/Default/Default.types.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as bs from 'react-bootstrap';
+
+export interface DropDownItemProps {
+  id: string;
+  label: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: any | undefined;
+  length: number;
+}
+
+export interface DropDownContainerProps {
+  label: string;
+  selectedItem: string;
+  items: DropDownItemProps;
```

Comment:
I believe we could stick to the `Dropdown` naming convention, there are some instances of `Dropdown` and `DropDown`. Could you rename everything to `Dropdown`?